### PR TITLE
Add tokenizer to encode input text and decode predicted image from logits

### PR DESCRIPTION
**Pull request by mahalrs**
Tokenizer to encode text using BartTokenizer. It decodes Bart decoder output of image tokens to image using VQGAN decoder.

### CODE REVIEW

**Comment by ArjunKrishna3367** [comment]
Currently getting an error when trying the tokenizer:
‘BartTokenizer’ object has no attribute ‘to’, for running the line tokenizer.to(device)

Not working with cpu nor cuda.
Trying to see if there's an easy fix.

**Comment by mahalrs** [comment]
Let me fix it

**Review by ArjunKrishna3367 — APPROVED** [review]
Looks good, was able to get the new tokenizer running on both images and text in the notebook.
